### PR TITLE
Clean up Table Understanding notebook

### PR DESCRIPTION
**Pull request by frreiss**
This PR takes the first part of the notebook that was at `notebooks/Tables_data.ipynb` and adds explanatory text, plus some additional data cleaning code. The results of this change are now in a new notebook called `notebooks/Understand_Tables.ipynb`.

I've kept the original notebook around, but moved it to `notebooks/old` for now, because there is some additional content at the end of the original file that isn't yet in a new notebook. We should migrate that code, which runs through all the tables in 2 years of IBM financial reports, into a new notebook that shows all the types of financial table that our table processing can convert to Pandas.

### CODE REVIEW

**Comment by frreiss** [comment]
No idea what's up with Travis. The builds marked "queued" have logs of successful runs attached to them ¯\_(ツ)_/¯

File: notebooks/Understand_Tables.ipynb [hunk 1]
```diff
@@ -0,0 +1,4387 @@
+{
```
Comment (BryanCutler) [inline notebooks/Understand_Tables.ipynb:1]: <p>"occurred under under" -&gt; "occurred under"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (BryanCutler) [inline notebooks/Understand_Tables.ipynb:1]: <p>wrap as a series to display better?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (BryanCutler) [inline notebooks/Understand_Tables.ipynb:1]: <p>So this extra space was due to a bug in Watson Table Understanding?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

**Comment by BryanCutler** [comment]
Looks great, this should be a really useful addition!

File: notebooks/Understand_Tables.ipynb (hunk 1 above)
Comment (frreiss) [reply notebooks/Understand_Tables.ipynb:1]: Fixed now.

Comment (frreiss) [reply notebooks/Understand_Tables.ipynb:1]: <p>Replaced with <code>drop_duplicates()</code>  on a one-column DataFrame. Displays better now.</p>

Comment (frreiss) [reply notebooks/Understand_Tables.ipynb:1]: <p>Technically it's a bug in the PDF-&gt;HTML conversion upstream of Watson Table Understanding. Most likely the PDF document kerned the "i" closer to the "f" than to the "c", and the conversion software mistook the difference in spacing as a space after the "c".</p>

Comment (MLnick) [inline notebooks/Understand_Tables.ipynb:1]: <p>Minor nit - but I think there's a space before "Understand"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (MLnick) [inline notebooks/Understand_Tables.ipynb:1]: <ul><li>"Here's that table looks like close up: " - "Here's what that table looks like close up:"</li><li>Just wondering if we should include a (short) sentence describing what Watson Discovery is (and perhaps hyperlink also)?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by MLnick** [comment]
Made a couple very minor nit comments. Looks great! Good intro / setting out the problem, and good flow through the details.

File: notebooks/Understand_Tables.ipynb (hunk 1 above)
Comment (frreiss) [reply notebooks/Understand_Tables.ipynb:1]: <p>Tightened up the HTML at the top of this cell.</p>

Comment (frreiss) [reply notebooks/Understand_Tables.ipynb:1]: <p>Good idea! I've tweaked the intro a bit to include a more direct reference to Watson Discovery.</p>

**Comment by frreiss** [comment]
Thanks for the review, @MLnick ! I've pushed some additional improvements to the notebook. Will merge once CI completes.

**Pull request merged**